src/ape/api/query.py

```diff
@@ -69,9 +69,23 @@ def all_fields(cls) -> List[str]:
         return list(BlockAPI.__fields__)
 
 
-class _BaseAccountQuery(_BaseQuery):
+class BlockTransactionQuery(_BaseQuery):
+    block_id: Any
+
+    @classmethod
+    def all_fields(cls) -> List[str]:
+        return list(TransactionAPI.__fields__)
+
+
+class AccountTransactionQuery(_BaseQuery):
+    """
+    A ``QueryType`` that collects properties of ``TransactionAPI`` over a range
+    of transactions made by ``account`` between ``start_nonce`` and ``stop_nonce``.
+    """
+
     start_nonce: NonNegativeInt = 0
     stop_nonce: NonNegativeInt
+    account: AddressType
```

Comment:
Just want reverse the ordering a bit here:
```suggestion
    account: AddressType
    start_nonce: NonNegativeInt = 0
    stop_nonce: NonNegativeInt
```